azure-pipelines.perf-test.yml

```diff
@@ -10,86 +10,132 @@ variables:
+
+      # Build @fluentui/perf-test-northstar
+      - script: |
+          yarn lage build --to @fluentui/perf-test-northstar --to @fluentui/react-conformance --verbose
+        condition: and(variables.PackageAffected, variables.isPR)
+        displayName: Build to Perf Test (Northstar)
+
+      # @fluentui/perf-test-northstar Baseline
+      - script: |
+          yarn perf:test:base
+        condition: and(variables.PackageAffected, eq(variables.isPR, false))
+        workingDirectory: packages/fluentui/perf-test-northstar
+        displayName: Run Perf Test Base (Northstar)
+
+      # Run @fluentui/perf-test-northstar
+      - script: |
+          yarn perf:test
+        condition: and(variables.PackageAffected, variables.isPR)
+        workingDirectory: packages/fluentui/perf-test-northstar
+        displayName: Run Perf Test (Northstar)
+
+      - task: AzureUpload@2
+        condition: and(variables.PackageAffected, variables.isPR)
+        displayName: Upload Perf Test Result to PR deploy site (Fluent N*)
+        inputs:
+          SourcePath: 'packages/fluentui/perf-test-northstar/dist'
+          azureSubscription: $(azureSubscription)
+          storage: $(azureStorage)
+          ContainerName: '$web'
+          BlobPrefix: '$(deployBasePath)/perf-test-northstar'
+
+      - task: GithubPRComment@0
+        condition: and(variables.PackageAffected, variables.isPR)
+        displayName: 'Post @fluentui/react-northstar Perf Results to Github Pull Request'
+        inputs:
+          githubOwner: microsoft
+          githubRepo: 'fluentui'
+          blobFilePath: '$(Build.SourcesDirectory)/$(PerfCommentFilePathNorthstar)'
+          status: '$(PerfCommentStatusNorthstar)'
+          uniqueId: 'perfComment9424'
+
+      - script: |
+          yarn stats:build
+        condition: and(variables.PackageAffected, eq(variables.isPR, false))
+        displayName: Bundle Statistics (master only)
+
+      - script: |
+          yarn perf
+        condition: and(variables.PackageAffected, eq(variables.isPR, false))
+        displayName: Performance Tests (master only)
+
+      # HEADS UP: also see tag-version-prefix in fluentui-publish.js
+      - script: |
+          yarn stats:save --tag=`git tag --points-at HEAD | grep ^@fluentui/react-northstar_v | grep -o 'northstar_v.*'`
+        condition: and(variables.PackageAffected, eq(variables.isPR, false))
+        displayName: Save Statistics to DB (master only)
+        env:
+          STATS_URI: $(STATS_URI)
+
+      - template: .devops/templates/cleanup.yml
+
+  - job: PerfTestFabric
+    displayName: Perf test for @fluentui/react
+    workspace:
+      clean: all
+    steps:
+      - template: .devops/templates/tools.yml
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      - script: |
+          yarn just check-if-package-affected --packageAffected @fluentui/react --packageAffected @fluentui/react-components
```

Comment:
Should we be splitting v8 and v9 as well?

Reply:
Yup, going to do that in a separate PR once this one gets checked in